glib/src/auto/match_info.rs

```diff
@@ -0,0 +1,91 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::{translate::*, Error, Regex};
+
+crate::wrapper! {
+    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct MatchInfo(Shared<ffi::GMatchInfo>);
+
+    match fn {
+        ref => |ptr| ffi::g_match_info_ref(ptr),
+        unref => |ptr| ffi::g_match_info_unref(ptr),
+        type_ => || ffi::g_match_info_get_type(),
+    }
+}
+
+impl MatchInfo {
+    #[doc(alias = "g_match_info_fetch")]
+    pub fn fetch(&self, match_num: i32) -> Option<crate::GString> {
+        unsafe { from_glib_full(ffi::g_match_info_fetch(self.to_glib_none().0, match_num)) }
+    }
+
+    #[doc(alias = "g_match_info_fetch_all")]
+    pub fn fetch_all(&self) -> Vec<crate::GString> {
```

Comment:
```suggestion
    pub fn fetch_all(&self) -> glib::PtrSlice<crate::GString> {
```

Would avoid one unnecessary copy (into the `Vec`) here